etc/install-cgi/test.gw

```diff
@@ -9,6 +9,16 @@ beg
 - f Tiny 1/1/2022 ?
 end
 
+notes MOUSE Tiny
+beg
+The picture above has historically been a test image for compression and image rendering. Its story can be read in <a href="http://www.lenna.org">lenna.org</a>.
+
+The full picture of lenna (nudity!):
+<a href="%sm=IM&s=Lenna-full.jpg">
+<img src="%sm=IM&s=Lenna-icon.jpg">
+</a>
+end notes
```

Comment:
Don't you think this might offend some users, namely members of the church of Jesus Christ of Latter-day Saints?

Reply:
Do you think they open the source of Geneweb? C'est de la recherche. Henri a utilisé sa femme aussi pour la Science. À cause de toi j'ai cliqué sur « full picture », zut alors un corps humain comme il sort à l'impression dès la naissance, avec des bottes et un chapeau. Merci madame pour la Science ! Ce ne sont pas des Amish les Mormons, je te laisse lire ça https://www.madmoizelle.com/pour-coucher-sans-pecher-les-ados-mormons-inventent-des-techniques-qui-fascinent-tiktok-1199241 que je découvre grâce à toi également, c'est… comment dire… surprenant !

Reply:
Allez, je vais me contenter de laisser le lien vers l'histoire de Lenna où les gens trouveront la même image s'ils sont curieux.

Reply:
Il faudra peut etre ajouter un type de temoin en plus pour specifier la personne sautant sur le lit lors de la conception d'un enfant

Reply:
J'adore ce commentaire ! Par contre, plus sérieusement, on a viré (complètement ?) les fonctions de « multi-parentalité » qu'avait mis en place Daniel à la fin de la v5. Ce n'était pas terminé, Maufred n’a pas entretenu ni poussé la chose, ça n’était pas utilisé par le front-end réellement et je ne sais pas dire quant au stockage dans la base format gw, encore moins si c'était exportable en .ged ensuite ! L’idée en fond était louable d’autant que c’est maintenant réel¹ qu’il y a des enfants issus de trois ADNs différents, ça existe. Ça rejoint un peu notre mauvaise gestion des genres il en faudrait au moins un 4e pour neutre qui est différent d’inconnu. Et surtout qu'on ne gère pas du tout le changement de sexe comme événement possible. Si un wokiste lit ça je pense qu'il me trucide pour le vocabulaire utilisé. Tout ceci dit, je pense que ça n'a plus d’importance, il me semble que Sagotch avait viré le besoin de vérifier le sexe pour les mariages (nschk)… enfin j'ai pas ressayé depuis mais quand je fixbase sur ma base perso, je vois passer mes quelques couples homosexuels à chaque fois ce qui m'y refait penser. Et pareil, on a du mal à « saisir » correctement les diverses PMA quand don de sperme ou d’ovocyte qui sont de plus en plus fréquentes quand y'a des problèmes de fertilité dans le couple. Et légalement comme pour l'adoption la mère ou père deviennent les parents mais ne sont pas les géniteurs biologiques… Et ça on ne sait jamais le transcrire comme il faut. On pourrait imagine un « “mère” l'ayant porté » dans les relations, mais c'est un vocabulaire que je ne maîtrise guère. Ceci dit c'est clairement un besoin croissant !!

¹ https://www.lemonde.fr/sciences/article/2016/09/28/premiere-naissance-d-un-bebe-a-trois-parents_5004492_1650684.html

Reply:
Au minimum, il faut vérifier ce qu'on fait dans les cas de couples homosexuels pour lesquels LUI/ELLE ne convient plus. Il y a un P1/P2 plus ou moins en place, mais peut être pas systématiquement.
Pour la multi-parentalité, j'ai peur que ce soit par trop embryonnaire dans GeneWeb, et qu'on ne sache pas vraiment ce qu'on veut faire.